doc/source/serve/doc_code/model_composition/streaming_example.py

```diff
@@ -0,0 +1,44 @@
+# flake8: noqa
+# __streaming_example_start__
+# File name: stream.py
+from typing import AsyncGenerator, Generator
+
+from ray import serve
+from ray.serve.handle import DeploymentHandle, DeploymentResponseGenerator
+
+
+@serve.deployment
+class Streamer:
```

Comment:
possible to use a real language model example here?

Reply:
we already have that in the streaming guide, this is just showing how to use the handle API